src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql

```diff
@@ -506,4 +506,456 @@ select
+           ,e.region as enrollment_region
+           ,e.school_level as enrollment_school_level
+           ,e.schoolid as enrollment_school_id
+           ,e.school_name as enrollment_school
+           ,e.school_abbreviation as enrollment_school_abbreviation
+           ,e.studentid as student_id_enrollment
+           ,e.student_number as student_number_enrollment
+           ,e.lastfirst as student_name
+           ,e.first_name as student_first_name
+           ,e.last_name as student_last_name
+           ,case when cast(e.grade_level as string) = '0' then 'K'
+                 else cast(e.grade_level as string) end as enrollment_grade_level
+           ,case when is_out_of_district = true then 1
+                 else 0 end as ood
+           ,e.gender as gender
+           ,e.ethnicity as ethnicity
+           ,case when e.is_homeless = true then 1
+                 else 0 end as homeless
+           ,case when e.is_504 = true then 1
+                 else 0 end as is_504
+           ,case when e.spedlep in ('no iEp',null) then 0
+                      else 1 end as sped
+           ,case when e.lep_status = true then 1
+                 else 0 end as lep
+           ,lunch_application_status as lunch_status
+     from {{ ref("base_powerschool__student_enrollments") }} as e
+     where academic_year = 2022 and enroll_status = 0 and rn_year = 1 and grade_level <= 4
+),
+
+schedule as
+(
+      select c._dbt_source_relation
+            ,cast(c.cc_academic_year as string) as schedule_academic_year
+            ,'Kipp nJ/miami' as schedule_district
+            ,e.region as schedule_region
+            ,e.schedule_school_abbreviation
+            ,c.cc_schoolid as schedule_school_id
+            ,e.school_name as schedule_school
+            ,c.cc_studentid as student_id_schedule
+            ,e.student_number_enrollment as student_number_schedule --needed to keep track of students who test at a grade level different from the class they are enrolled at
+            ,case when c.courses_course_name in ('ela grK','ela K') then 'K'
+                  when c.courses_course_name = 'ela gr1' then '1'
+                  when c.courses_course_name = 'ela gr2' then '2'
+                  when c.courses_course_name = 'ela gr3' then '3'
+                  when c.courses_course_name = 'ela gr4' then '4'
+                  end as schedule_student_grade_level
+            ,c.cc_teacherid as teacher_id
+            ,c.teacher_lastfirst as teacher_number
+            ,c.teacher_lastfirst as teacher_name
+            ,c.courses_course_name as course_name
+            ,c.cc_course_number as course_number
+            ,c.cc_section_number as section_number
+
+            ,period as expected_test
+      from {{ ref("base_powerschool__course_enrollments") }} as c
+      left join --this join is needed to bring over the student_number onto the scheduling table, as mclass data contains only the student number field
+      (

```

Comment:
don't join to a subquery:
- refactor this to a CTE

Reply:
this is also the same query as the student CTE above:
- left join to the CTE instead